### DISCUSSION TITLE
v.0.12.1

### DISCUSSION THREAD

**Discussion by DavidLazarescu** [Librum-Reader/Librum#145] [Announcements]
- Added Arabic translation and support for RTL languages
- Added 'Open With Librum' support for Windows and Linux
- Changed to only using 'Name' instead of first and last name
- Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/Librum-Reader/Librum/releases/tag/v.0.12.1'>v.0.12.1</a>.</em>

**Comment by arul-intcygen** [comment]
is this really how the app works. but whenever i open librum, i need to reinstall all books i wanna read even though the books are already saved in local directory.

**Reply by DavidLazarescu** [reply]
Hey, do you mean that you need to redownload books that you have already downloaded (by clicking on them and loading their files)? This should not happen. Librum will fetch your non-dowloaded books (the ones which only exist in your online library) on every application start but if you have downloaded a book, it should be on your PC until you delete it.

**Reply by arul-intcygen** [reply]
Yes, that's what happened on my linux system. It's a shame that I face a problem while using this very cool ebook reader. I will recommend Librum to my friends so I know whether this case only occurs on my PC or not or perhaps by reinstalling Librum itself i'll get the answer. Really appreciate your hard work, and your reply. Thank you.

**Reply by DavidLazarescu** [reply]
I am sorry to hear that. It would be great if you could send me the log file (~/.local/share/Librum-Reader/Librum/librum_log.txt on Linux) so that I can check if there is any issue on that end.

I have never heard of such a problem, so i hope it's a one time thing. If this happens again please text me here or at contact@librumreader.com so that I can investigate further)

**Reply by arul-intcygen** [reply]
sure thing, thanks in advance

**Comment by Audiosutras** [comment]
@DavidLazarescu, I recently opened Librum with the `0.12.1` update. It looks like Librum is requiring an account be created for data persistence in the application? Is this correct? What is the reasoning here or is there a setting that needs to enabled to use your local filesystem? I'm a little unhappy by this change.

**Reply by DavidLazarescu** [reply]
Hey, Librum always required an account for data persistence. 

The idea behind Librum is to be an online library with features like folders, ai tooling, statistics which all require an account.

You can self host Librum and avoid syncing your data to the official servers, but you need an account.

**Reply by Audiosutras** [reply]
Thanks for bringing clarity to this topic!
